api/list.js

```diff
@@ -1,7 +1,6 @@
 const metadata = require("../util/metadata");
 
-// return an array of expressjs callbacks, the first using multer to support
-// uploading multipart forms (ie, files), and the second to handle extraction
+// Return a function for getting list of deployed spas and info about them
```

Comment:
Many thanks, I missed this comment. Clearly I copied the code from /deploy initially.